src/app/projects/project-details/project-details.component.html

```diff
@@ -65,9 +73,8 @@ <h4>{{project.name}}</h4>
   </ng-template>
 
   <ng-template #configuration>
-    <wh-project-details-configuration
-      [project]="project"
-      (wantToRefresh)="refreshProject()"></wh-project-details-configuration>
+    <wh-project-details-configuration [project]="project"
+                                      (wantToRefresh)="refreshProject()"></wh-project-details-configuration>
```

Comment:
If I see correctly, then you did nothing but format this file. Suggest reverting.

If you insist on formatting it this way, then as we don't have a strict formatting styleguide for the HTML yet, I suggest we discuss something in the team and come to a conclusion, followed by a separate PR for applying that formatting. But until then, could you make your formatter less obtrusive?

It's polluting the diffs a lot. And the "Hide whitespace changes" seems to disable the syntax coloring :(